tests/unit/services/ember-cli-test.js

```diff
@@ -65,6 +65,21 @@ test("buildProperties() works as expected with replacements", function (assert)
   assert.ok(props.content.indexOf('helper(myHelper)') !== -1, 'Replacements worked if multiple');
 });
 
+test("buildProperties('model') creates a valid file for a model", function(assert) {
```

Comment:
So you're suggesting not adding a test at all? I wouldn't expect too many changes for `blueprints/model.js` since all the basic stuff which is required to start implementing a model is there.

Just wanted to check before I remove the test...